sched_boundary/sched_boundary.py

```diff
@@ -143,6 +148,7 @@ def function_define(self, decl, _):
                 "params": ", ".join(GccBugs.fix(arg, arg.type.str_no_uid) \
                         for arg in decl.arguments) if decl.arguments else "void"
             }
+            GccBugs.handle3dots(decl, signature)
```

Comment:
It's better to export only one API (GccBugs.fix) from GccBugs class. I don't like the current design.

Reply:
or even pack line 146-157